<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>蘆洲慢跑會員系統</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="TabMemberInfo">
     <attribute name="title">
      <string>會員資料</string>
     </attribute>
     <widget class="QFrame" name="frame_mbr_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl_mbr_photo">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>20</y>
         <width>200</width>
         <height>250</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>340</y>
         <width>176</width>
         <height>66</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_mbr_info_create">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_mbr_info_update">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>381</width>
         <height>395</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_mbr_id">
          <property name="text">
           <string>會員編號</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edit_mbr_id">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_mbr_position">
          <property name="text">
           <string>職稱</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edit_mbr_position">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_mbr_group">
          <property name="text">
           <string>分組</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lbl_mbr_name">
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="edit_mbr_name">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lbl_mbr_idcard">
          <property name="text">
           <string>身分證</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="edit_mbr_idcard">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lbl_mbr_birth_ROC">
          <property name="text">
           <string>年次</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="edit_mbr_birth_ROC">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lbl_mbr_birthday">
          <property name="text">
           <string>生日</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="edit_mbr_birthday">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="lbl_mbr_phone1">
          <property name="text">
           <string>電話</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="edit_mbr_phone1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="lbl_mbr_phone2">
          <property name="text">
           <string>電話(2)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="edit_mbr_phone2">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="lbl_mbr_cellphone">
          <property name="text">
           <string>手機</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="edit_mbr_cellphone">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="lbl_mbr_area">
          <property name="text">
           <string>區域</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="edit_mbr_area">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="lbl_mbr_address">
          <property name="text">
           <string>通訊地址</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="edit_mbr_address">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edit_mbr_group">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="lbl_mbr_photo_path">
          <property name="text">
           <string>照片位置</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLineEdit" name="edit_mbr_photo_path">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>lbl_mbr_photo</zorder>
      <zorder>gridLayoutWidget_2</zorder>
      <zorder>formLayoutWidget</zorder>
      <zorder>btn_mbr_info_update</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="TabMemberTable">
     <attribute name="title">
      <string>會員清單</string>
     </attribute>
     <widget class="QFrame" name="frame_mbr_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>721</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="TabStatistic">
     <attribute name="title">
      <string>統計資料</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>分組名單</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>馬拉松志工調度</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionImport_from_excel"/>
    <addaction name="actionExport_to_excel"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_from_excel">
   <property name="text">
    <string>Import from excel</string>
   </property>
  </action>
  <action name="actionExport_to_excel">
   <property name="text">
    <string>Export to excel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
